packages/credentials/tests/presentation-exchange.spec.ts

```diff
@@ -209,6 +210,22 @@ describe('PresentationExchange', () => {
       expect(warnings?.length).to.equal(0);
     });
   });
+
+  describe('Web5TestVectorsPresentationExchangeSpec', () => {
+    it('select_credentials', async () => {
+      const jsonString = fs.readFileSync('../../test-vectors/presentation_exchange/select_credentials.json', 'utf8');
+      const vectors = JSON.parse(jsonString).vectors;
+
+      for (let i = 0; i < vectors.length; i++) {
+        const input = vectors[i].input;
+        const expectedOutput = vectors[i].output.selectedCredentials;
+
+        const selectedCreds = await PresentationExchange.selectCredentials({ vcJwts: input.credentialJwts, presentationDefinition: input.presentationDefinition});
```

Comment:
`PresentationExchange.selectCredentials` is not an async method:

```suggestion
        const selectedCreds = PresentationExchange.selectCredentials({ vcJwts: input.credentialJwts, presentationDefinition: input.presentationDefinition });
```